let promise  =  new Promise((reject,resolve)=>{
    
        setTimeout(()=>{}_)
)
promise.then((mess)=>
console.log(mess))
.catch((error)=>{
    console.log(error)
})
